_component_design/vertical-stepper.md

```diff
@@ -11,7 +11,7 @@ status: stable
 last-modified: 2018-08-21
 helix-ui-css: true
 helix-ui-javascript: true
-pagelink: https://rackerlabs.github.io/helix-ui/components/accordions/
+pagelink: components/accordion/
```

Comment:
Note: Technically, it's `components/stepper/`, but Stepper docs are in beta _and_ they need to be moved into the Accordion docs (as a styled variant) at some point.

Reply:
ok.  will hold for now